Runtime/VisibilityCondition.cs

```diff
@@ -0,0 +1,28 @@
+using UnityEngine;
+
+namespace Mixin.Utils
+{
+    /// <summary>
+    /// A component to either hide or show this object on a specific device.
+    /// </summary>
+    public class VisibilityCondition : MonoBehaviour
+    {
+        /// <summary>
+        /// Define the visibility of a specific platform.
+        /// </summary>
+        [Tooltip("Define the visibility of a specific platform.")]
+        [SerializeField]
+        private MixinDictionary<RuntimePlatform, bool> _platformVisibilityDictionary;
+
+        void Start()
+        {
+            // Get the current platform.
+            RuntimePlatform currentPlatform = Application.platform;
+
+            // Check if it contains the current platform.
+            if (_platformVisibilityDictionary.ContainsKey(currentPlatform))
+                // If so, then make the Object (in)visible.
+                gameObject.SetActive(_platformVisibilityDictionary[currentPlatform]);
```

Comment:
Maybe if no key is found, it should default to setting it invisible, instead of leaving it how it is.
Something like:


`bool makeVisible = false;`
`if (_platformVisibilityDictionary.ContainsKey(currentPlatform))`
`    makeVisible = _platformVisibilityDictionary[currentPlatform;`

`gameObject.SetActive(makeVisible);`